cassandra-migration/src/main/java/org/cognitor/cassandra/migration/MigrationRepository.java

```diff
@@ -77,6 +79,7 @@
     private final ScannerRegistry scannerRegistry;
     private List<ScriptFile> migrationScripts;
     private final ScriptCollector scriptCollector;
+    private final ScriptFilter scriptFilter;
```

Comment:
Can you make this field non final and assign the IdempotentFilter as the default? Then add a setter method so that the default behaviour can be changed. As this is currently something more optional I don't think we need to introduce another constructor parameter for this.

Reply:
done